src/data/markdown/docs/03 cloud/04 Integrations/05 Cloud APM/01 Azure Monitor.md

```diff
@@ -0,0 +1,156 @@
+| azureRegion           | The `azureRegion` you've created your Azure configurations. See the list of [supported regions](#supported-regions). Default is `eastus`.   |
+
+
+<br/>
+You can follow these instructions to get the required Azure Monitor settings:
+
+1. Make sure you have an active subscription on Azure. Copy the subscription ID for `subscriptionId` setting.
+
+    ![Azure subscriptions](images/azure-subscriptions.png)
+
+2. Create a resource group under your active subscription. Copy the name for the `resourceGroupName` setting.
+
+    ![Azure resource groups](images/azure-resource-groups.png)
+
+3. Create a service principal by adding a new app to Azure app registrations. Copy the name for the `tenantId` setting. 
+
+    ![Azure app registrations](images/azure-app-registrations.png)
+
+4. Create a client secret in your service principal. Copy the `clientId` and `clientSecret` settings.
+
+    ![Client secrets](images/client-secrets.png)
+
+5. Go to your resource group and assign "Monitoring Metrics Publisher" role to the user and the service principal.
+
+    ![Resource group's access controls](images/rg-access-controls.png)
+
+6. Create a log analytics workspace. The region is now created. Copy the region name for the `azureRegion` setting.
+
+    ![Log analytics workspace](images/azure-log-analytics-workspace.png)
+
+8. Create an app under application insights with your log analytics workspace. The region should match the log analytics workspace. Copy the app name for the `insightsAppName` setting.
+
+    ![Application insights](images/azure-application-insights.png)
+
+## k6 Cloud test configuration
+
+You have to enable the Azure Monitor integration for each test that you want to export its k6 metrics when running the test.
+
+Currently, there are two options to set up the Cloud APM settings in the test:
+
+- [Using the test builder](#configuration-using-the-test-builder)
+- [Scripting the k6 test](#configuration-in-the-k6-script)  
+
+### Configuration using the test builder
+
+First, you have to enable the Azure Monitor integration into your organization. Click the `Cloud APM` option on the left sidebar menu under the `Manage` section, and select `Azure Monitor` from the list.
+
+![Cloud APM - Azure Monitor Form UI](images/azure-monitor-cloud-app-form.png)
+
+In this form, set the parameters that you copied in the previous steps.  For more information on the other input fields, see [configuration parameters](#configuration-parameters).
+
+Save the Azure Monitor configuration for the current organization. 
+
+Now, you can use the [test builder](/test-authoring/test-builder) to enable the integration for a new or existing test on the organization.
+
+![Cloud APM - Azure Monitor Test Builder UI](images/azure-monitor-cloud-app-testbuilder.png)
+
+
+### Configuration in the k6 script
+
+If you script your k6 tests, you can also configure the Cloud APM settings using the `apm` option in the k6 script. 
```

Comment:
Maybe we could point out here that the user can also use the cloud app to create scripts? Something like:

"You can create new scripts in the cloud app by clicking `Create new test` button and selecting `Start scripting` option."

Reply:
I think that if we describe this particular method for scripting a test, we also need to describe the other scripting method: scripting with your IDE and run the cloud test with the CLI. 

I prefer covering all the cases with something like `script the k6 tests` to avoid enumerating the different scripting methods. 

If you have something in mind, please, feel free to give a try and share it.

Reply:
Gotcha, makes sense. 👍